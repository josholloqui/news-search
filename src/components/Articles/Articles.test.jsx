import React from 'react';
import { render, cleanup } from '@testing-library/react';
import ArticleList from './Articles';

describe('Article component', () => {
  afterEach(() => cleanup());
  it('renders Article', () => {
    const { asFragment } = render(
      <ArticleList
        articles={[
          {
            title: 'calvin',
            author: 'and',
            description: 'hobbs',
            publishedAt: 'today'
          }
        ]}
      />
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
