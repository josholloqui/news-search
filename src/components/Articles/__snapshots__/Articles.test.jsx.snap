// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article component renders Article 1`] = `
<DocumentFragment>
  <ul
    data-testid="articles"
  >
    <li>
      <h3>
        calvin
      </h3>
      <cite>
        By: and
      </cite>
      <p>
        Description: hobbs
      </p>
    </li>
  </ul>
</DocumentFragment>
`;
